rdr_service/resource/generators/genomics.py

```diff
@@ -92,6 +92,12 @@ def make_resource(self, _pk, backup=False):
             row = ro_session.execute(text('select * from genomic_set_member where id = :id'), {'id': _pk}).first()
             data = self.ro_dao.to_dict(row)
 
+            # Delete biobank_id key since we send to biobank_id_str
+            try:
+                del data['biobank_id']
+            except KeyError:
+                pass
+
```

Comment:
Does this not need to set the `biobank_id_str` field?

Reply:
Yes, I missed that here. Updating now.